docs/docsource/installation/upgrade/upgrade_zenko.rst

```diff
@@ -87,3 +87,34 @@ To upgrade Zenko:
       Expect the cluster to take a few minutes to stabilize. You may see 
       CrashLoopBackoff errors. This is normal, expected behavior.
 
+#. With the introduction of Helm 3 support of Helm 2 charts are still 
+   supported. Helm 3 charts introduces improved functionality with support for 
+   existing Helm 2 charts and in place upgrades and conversions.
```

Comment:
```suggestion
#. With the introduction of Helm 3, Helm 2 charts are still 
   supported. Helm 3 charts introduce improved functionality, and support
   existing Helm 2 charts and in-place upgrades and conversion.
```

Reply:
This is not a continuation of the previous process. It introduces a new process (numbered, below)